backend/src/pilstark/json_exporter/mod.rs

```diff
@@ -61,7 +61,7 @@ pub fn export<T: FieldElement>(analyzed: &Analyzed<T>) -> PIL {
                 let (_, expr) = exporter.polynomial_reference_to_json(&AlgebraicReference {
                     name: pub_ref.name.clone(),
                     poly_id: pub_ref.poly_id.unwrap(),
-                    index: pub_ref.index,
+                    index: None,
```

Comment:
What happens to arrays here?

Reply:
This might actually be a mistake since index on PublicDeclaration does not refer to index access but to the row number. Let me check.

Reply:
What's the conclusion here?